### PR TITLE
Hotfix for lingseeks

### PR DESCRIPTION
**Pull request by Koriath**
Missed a RETURN there.
Also a tiny solar tweak.

### CODE REVIEW

**Comment by Koriath** [comment]
Alright, now works as intended, tested and includes spooky sounds too.

**Comment by The-Albinobigfoot** [comment]
Is the "meeseeks/gibber" bug fixed yet, last week I attempted to gib meeseeks multiple times, and they all jammed the gibber.

**Comment by Koriath** [comment]
Not sure what caused that, I'll check the gibber code.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -445,10 +454,12 @@
...
 		max_brain_damage = 80
 		stage_counter = 1 //to stop the spam of "I CAN'T TAKE IT"
-	var/mob/living/carbon/human/MST = master
 
+	var/mob/living/carbon/human/MST = master
 	if((MST && MST.stat == DEAD) || !MST)
-		if(findtextEx(H.real_name, "Mr. Meeseeks (") == 0 && !lingseek) // This mob has no business being a meeseeks // AHAH, now it's a Lingseeks!
+		if(lingseek)
+			return //everything is fine
+		else if(findtextEx(H.real_name, "Mr. Meeseeks (") == 0 && !lingseek) // This mob has no business being a meeseeks // AHAH, now it's a Lingseeks!
```
Comment (octareenroon91) [inline code/modules/mob/living/carbon/human/species_types.dm]: Mutating into someone else gives you their real_name .  This check was only valid for staff-of-change transformation.

**Comment by octareenroon91** [comment]
Pulling and patching on top of it.

**Pull request merged**